scripts/generate-configs

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash ../shell.nix
```

Comment:
Is `nix-shell` smart enough to resolve this relative to the script, and not to the working directory?

Reply:
It resolves relative to the script location - testing that out did reveal that `cabal` needs to be added to the `buildInputs`, though.